cmd/node/config/gasSchedules/gasScheduleV1.toml

```diff
@@ -691,3 +691,16 @@
     MaxBuiltInCallsPerTx = 100
     MaxNumberOfTransfersPerTx = 100
     MaxNumberOfTrieReadsPerTx = 100
+
+# Quadratic, Linear and Constant are the coefficients for a quadratic func. Separate variables are used for the
+# sign of each coefficient, 0 meaning + and 1 meaning -
```

Comment:
`0 meaning positive and 1 meaning negative`?
as it reads better. Valid on all the other gasmaps